### DISCUSSION TITLE
v2.11.0

### DISCUSSION THREAD

**Discussion by zkoppert** [github-community-projects/issue-metrics#143] [Announcements]
# Changelog
## 🚀 Features

- Support more than one statistic for summarizing time lists @MaineC (#127)

## 🧰 Maintenance

- [docs] Add back references to footnotes @spier (#141)
- Fix README collisions @zkoppert (#140)
- docs: update README (take over #131) @Okabe-Junya (#139)
- Add dev dependencies to requirements.txt @zkoppert (#138)
- Docs improvements in README.md @spier (#137)
- build(deps): bump python from 3.11-slim to 3.12-slim @dependabot (#134)
- Support python 3.12 @zkoppert (#135)

See details of [all code changes](https://github.com/github/issue-metrics/compare/v2.10.0...v2.11.0) since previous release 

<hr /><em>This discussion was created from the release <a href='https://github.com/github/issue-metrics/releases/tag/v2.11.0'>v2.11.0</a>.</em>

**Comment by spier** [comment]
This is interesting @zkoppert! 

What's the benefit of having a discussion for each release, over just having the release (e.g. [this](https://github.com/github/issue-metrics/releases/tag/v2.11.0)) itself?

Is it mostly to foster conversation around the changes in the given release?

**Reply by zkoppert** [reply]
Yes! I like it because folks can comment on a certain release and ask a question or comment on it. I feel like this gives folks an avenue to comment in a relevant place instead of commenting on a closed issue or pull request which often gets ignored since its closed.
